libs/contact/tsconfig.spec.json

```diff
@@ -7,6 +7,7 @@
       "node"
     ]
   },
+  "emitDecoratorMetadata": true,
```

Comment:
don't think you need this since it exists in the tsconfig file that this file extends

Reply:
8e2c34ad64c8c3ed5ce65740228937021fab22d0